app/test_main.py

```
@@ -1,3 +1,11 @@
 from app.main import get_human_age
 
-# write your code here
+def test_check_if_test_work_properly() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+    assert get_human_age(14, 14) == [0, 0]
+    assert get_human_age(15, 15) == [1, 1]
+    assert get_human_age(23, 23) == [1, 1]
+    assert get_human_age(24, 24) == [2, 2]
+    assert get_human_age(27, 27) == [2, 2]
+    assert get_human_age(28, 28) == [3, 2]
+    assert get_human_age(100, 100) == [21, 17]
```

This test function is missing a couple of requirements from the checklist:

1.  This violates checklist item #Clean Code.3: 'Use ```@pytest.mark.parametrize```'. Instead of having a single test function with many assertions, you should use parameterization to test multiple cases with a single test function structure. This makes tests more maintainable and provides better feedback on failures.
2.  This violates checklist item #Clean Code.2. The tests are missing checks for edge cases. You should test how the function behaves with negative numbers and what happens when it receives incorrect data types (e.g., `str`, `float`), which should raise an exception.